lib/models/obms.js

```diff
@@ -0,0 +1,183 @@
+// Copyrighr 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = ObmsModelFactory;
+
+ObmsModelFactory.$provide = 'Models.Obms';
+ObmsModelFactory.$inject = [
+    'Model',
+    'Services.Waterline',
+    '_',
+    'Services.Encryption',
+    'Constants',
+    'Promise',
+    'util',
+    'Errors',
+    'Assert'
+];
+
+function ObmsModelFactory (
+    Model,
+    waterline,
+    _,
+    encryption,
+    Constants,
+    Promise,
+    util,
+    Errors,
+    assert
+) {
+    var secretsPattern = _.reduce(Constants.Logging.Redactions, function(pattern, regex) {
```

Comment:
Actually the `Redactions` can be either regex or string, the string type redaction means "full word matching", check this code: https://github.com/RackHD/on-core/blob/master/lib/serializables/log-event.js#L149-L161

I think the best method is to move the Redaction to a common service function, so that it can be leveraged by other modules.  I can accept to do it with another PR.